recipe/meta.yaml

```diff
@@ -42,6 +46,10 @@ requirements:
         - libnetcdf
         - kealib
         - krb5
+        - libtiff
+        - libpng
+        - zlib  # [not win]
+        - jpeg  # [not win]
```

Comment:
Yes - but under Windows GDAL cannot be build against an external zlib for some reason (there is no option to do this). jpeg isn't available as a DLL under Windows, and the provided .lib is built for use in a .exe rather than a .dll so it won't even link to GDAL (another conda-forge project??)